managing-os/templates.md

```diff
@@ -125,21 +125,20 @@ Important Notes (R4.0)
    is always independent from its parent TemplateVM's `/home`, which means that any
    subsequent changes to the parent TemplateVM's `/home` will not affect
    the child TemplateBasedVM's `/home`.
-   
- * Template VMs are created in a thin pool, making `qvm-trim-template`
-   no longer necessary.
 
-   The root filesystems in Standalone VMs can employ
-   TRIM/discard on the root fs using normal tools and configuration options.
+ * All VMs are created in a thin pool by default, making `qvm-trim-template`
+   no longer necessary. You may wish to enable [TRIM/discard](/doc/disk-trim/)
+   in dom0, or use `fstrim` in dom0 monthly, but this will not impact VM free
+   space or disk usage.
```

Comment:
This just raises more questions than it answers. A 4.0 user who reads this probably wants to know how to accomplish the same thing as `qvm-trim-template` did on 3.2. This mentions two things that _don't_ affect free space or disk usage. Well, what _does_?

And, naturally, users are going to wonder: "If enabling TRIM/discard doesn't affect free space or disk usage, then what the heck does it do?!"

Reply:
> This just raises more questions than it answers.

You're right.

> A 4.0 user who reads this probably wants to know how to accomplish the same thing as `qvm-trim-template` did on 3.2.

One major benefit of using thin provisioning is that you don't need to manually trim things anymore. I'm not sure we should explain what thin provisioning is on this page. If it matters to the user there are plenty of LVM resources online.

> This mentions two things that _don't_ affect free space or disk usage.

Mentioning Disk TRIM doesn't make sense given the page's context, so I'll remove that part.

> Well, what _does_?

Storage allocation of thinly provisioned LVM pools is automatically handled, so no user action is needed.

> And, naturally, users are going to wonder: "If enabling TRIM/discard doesn't affect free space or disk usage, then what the heck does it do?!"

If anything, that should be explained in the TRIM article, or a simple search will find good resources--to keep the TRIM article Qubes-related. But, as I say above, I think I'll remove that part.